content/docs/user-guide/external-dependencies.md

```diff
@@ -243,3 +230,24 @@ The `url` and `rev_lock` subfields under `repo` are used to save the origin and
 [version](https://git-scm.com/docs/revisions) of the dependency, respectively.
 
 </details>
+
+## Example: DVC remote aliases
+
+If instead of a URL you'd like to use an alias that can be managed
```

Comment:
should we explain why would someone want to manage them independently?

Reply:
Sure. But I'm not sure what the answer is. Let me think about that.

> p.s. I only moved this section, it's not new.

Reply:
OK, done in 1e75862e, check it out when you can. Thanks